src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => (
+    century ? person.sex === 'm'
+    && Math.ceil(person.died / 100) === century : person.sex === 'm'
+  ));
+
+  const ageSum = men.reduce((prevMan, currMan) =>
+    prevMan + (currMan.died - currMan.born), 0);
```

Comment:
You don't need to use curr, it's self explanatory that its a current value
```suggestion
  const ageSum = men.reduce((sum, man) =>
    prevMan + (currMan.died - currMan.born), 0);
```